### ISSUE TITLE
J.R.R. Tolkien's The Lord of the Rings Vol 2 (1991)

### ISSUE THREAD

**Issue by TimberlAndRE-de** [exoscoriae/eXoDOS#3955]
in the manual there's a hint to transfer characters from the first part ( _J.R.R. Tolkien's The Lord of the Rings Vol 1 (1990)_ ) to this game.

```
Converting Characters from Lord of the Rings, Volume One to The Two Towers.

There is a program called CONVERT.EXE which will convert characters from Lord of the Rings, Volume One to The Two Towers. 
First, copy either DATAl.BIN or DATA2.BIN to the TOWERS directory. Then, type CONVERT
This will convert your files to The Two Towers.
```

I guess this could be added to the run.bat as a feature?

**Comment by exoscoriae** [comment]
ok... not sure how to test this. I added the feature to check both the cd and floppy version of part 1 for a save game. It is then smart enough to detect if it is there or not.

convert runs silent and gives no message, but it does update a file called NNPCS.DAT when you run convert with either DATA1.BIN or DATA2.bin present.

I started part 2, and then selected old game, and it said there were no saves.

So I picked new game, and it started a new game. What I'm not clear on, is if the new game included my characters from part 1. Presumably, I would need to actually get my Part 1 characters leveled up or with good items or something, and then start part 2 and see if I had any of that stuff. Someone is welcome to test this, but I believe I have done all I can without playing the game for the next few hours.
